astro/src/content/quickstarts/quickstart-swift-ios-native.mdx

```diff
@@ -0,0 +1,301 @@
+</Aside>
+
+FusionAuth will be initially configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+### Create Your iOS App
+
+Now you will create an iOS app. While this section builds a simple iOS app using [AppAuth](https://github.com/openid/AppAuth-ios), you can use the same configuration to integrate your existing app with FusionAuth.
+
+Open Xcode and click <Breadcrumb>File -> New Project</Breadcrumb>. Choose "iOS" as the platform, and "App" as the application type. Click <Uielement label="Next"/>.
+
+Set <Field label="Product Name"/> to `ChangeBank`. Set the <Field label="Organization Identifier"/> to `FusionAuth`.
+
+<Aside type="note">
+  If you use different values here, you will need to update the redirect URLs in your FusionAuth application to match your changes.
+</Aside>
+
+Click <Uielement label="Next"/>, and choose a location on your computer to save the project to.
+
+Now you have a standard "Hello World" app. You can run it in the simulator by clicking the play button in the top left corner of Xcode or selecting <Breadcrumb>Product -> Run</Breadcrumb> from the menu bar.
+
+## Authentication
+
+You'll use the [AppAuth Library](https://github.com/openid/AppAuth-ios), which simplifies integrating with FusionAuth and creating a secure web application.
+
+### Add The AppAuth Library
+
+There are many ways to add the AppAuth library to your project. The simplest way without installing any additional tools is to add the package to the project. 
+
+Select <Breadcrumb>File -> Add Package Dependency</Breadcrumb> from the menu bar. In the search bar, paste in `https://github.com/openid/AppAuth-ios`. In the list of packages select "appauth-ios" and click <Uielement label="Add Package"/>. You will be asked to choose the packages to install. On the "Add to Target" column select `Changebank` for both "AppAuth" and "AppAuthCore". Click <Uielement label="Add Package"/> to add the packages to your project.
+
+## Customization
+
+Before you start coding, you need to add some assets and styles to your project that will be referenced later.
+
+### Add Assets And Styles
+
+Add the ChangeBank logo to the project. Download it from [here](https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-swift-ios-native/main/complete-application/ChangeBank/Assets.xcassets/changebank.imageset/changebank.png). Click on the <Uielement label="Assets.xcassets"/> directory in the project navigator to open the asset manager. Drag and drop the downloaded `ChangeBank.png` file into the asset manager.
```

Comment:
Nitpick, but Assets.xcassets is just Assets in my XCode:
![ChangeBank](https://github.com/FusionAuth/fusionauth-site/assets/5517677/cc400c7f-ee92-4173-a047-2d4079587020)

Reply:
Additionally, file is all lower case `changebank.png`, and not PascalCase.

Reply:
Your preference is set to hide the file extensions thats why it is only showing the file name `Assets`, updated to only reflect the name